components/eda/code/eda-bbe/file.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Files, files as events?'
-description: Ballerina enhances file transfer with FTP listeners, reacting to events on servers. SFTP and FTPS options ensure security. Seamlessly integrate file transfer with Ballerina's secure functionality.
+description: Ballerina enhances file transfer with FTP listeners reacting to events on servers. SFTP and FTPS options ensure security. Seamlessly integrate file transfer with Ballerina's secure functionality.
```

Comment:
I feel the version with a comma is slightly clearer and more readable as it helps to separate the different components of the sentence more distinctly.

Reply:
Reverted to the original one.